src/Components/ErrorBoundary.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const ErrorBoundary = ({ children }) => {
```

Comment:
You should use [`<ErrorBoundary/>`](https://reactjs.org/docs/error-boundaries.html) component to catch possible errors

Reply:
fixed